sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -2271,7 +2271,11 @@ public void deleteCase(String caseUuid) {
 			throw new UnsupportedOperationException("User " + userService.getCurrentUser().getUuid() + " is not allowed to delete cases.");
 		}
 
-		caseService.delete(caseService.getByUuid(caseUuid));
+		Case caze = caseService.getByUuid(caseUuid);
+
+		externalJournalService.handleExternalJournalPersonUpdate(caze.getPerson().toReference());
+
+		caseService.delete(caze);
```

Comment:
Shouldn't the notify take place after the delete? It might not matter technically because the transaction is not over yet, but conceptually it makes sense.